packages/reactotron-core-server/src/reactotron-core-server.ts

```diff
@@ -112,17 +119,20 @@ export default class Server {
       this.emitter.emit("connect", partialConnection)
 
       // when this client disconnects
-      socket.on("disconnect", () => {
+      socket.on("close", () => {
         // remove them from the list partial list
         this.partialConnections = reject(
-          propEq("id", (socket as any).id),
+          propEq("id", thisConnectionId),
           this.partialConnections,
         ) as any
 
         // remove them from the main connections list
-        const severingConnection = find(propEq("id", (socket as any).id), this.connections)
+        const severingConnection = find(propEq("id", thisConnectionId), this.connections)
         if (severingConnection) {
-          (this.connections as any).remove(severingConnection)
```

Comment:
Not sure when `.remove` was there but its not now so had to do it another way.

Reply:
Oops.